README.md

```diff
@@ -70,9 +70,32 @@ adev scaffold contract 0xc939df369C0Fc240C975A6dEEEE77d87bCFaC259 beyond_pricer
 
 ## Installation
 
+*Install Python 3.10*
+- if on mac
 ```bash
-pip install autonomy-dev[all]
+brew install python@3.10
+export PATH="/opt/homebrew/opt/python@3.10/bin:$PATH"
+alias python=/opt/homebrew/opt/python@3.10/bin/python3.10
+alias pip=/opt/homebrew/opt/python@3.10/bin/pip3.10
+source ~/.zshrc
```

Comment:
There are tests in place to check the extracted commands from the docs.
This approach is quite assumptive for mac;

Have you heard of pyenv? It might make it much simpler to manage environments.